src/main/java/org/ow2/petals/flowable/FlowableSEConstants.java

```diff
@@ -349,6 +349,46 @@ private DBServer() {
     public static final File DEFAULT_IDM_ENGINE_CONFIGURATOR_CFG_FILE = null;
 
     /**
+     * Tag name of the value enbabling the Flowable REST API
+     */
+    public static final String ENGINE_REST_API_ENABLE = "engine-rest-api-enable";
+
+    /**
+     * Tag name of the group of users who are allowed to use the Flowable REST API
+     */
+    public static final String ENGINE_REST_API_ACCESS_GROUP = "engine-rest-api-access-group";
+
+    /**
+     * Tag name of the port number to access to the Flowable REST API
+     */
+    public static final String ENGINE_REST_API_PORT = "engine-rest-api-port";
+
+    /**
+     * Tag name of the Flowable REST API server address
+     */
+    public static final String ENGINE_REST_API_ADDRESS = "engine-rest-api-address";
+
+    /**
+     * Default value of {@link #DEFAULT_ENGINE_REST_API_PORT}.
+     */
+    public static final int DEFAULT_ENGINE_REST_API_PORT = 8089;
+
+    /**
+     * Default value of {@link #ENGINE_REST_API_ENABLE}
+     */
+    public static final boolean DEFAULT_ENGINE_REST_API_ENABLE = true;
+
+    /**
+     * Default value of {@link #ENGINE_REST_API_ACCESS_GROUP}
+     */
+    public static final String DEFAULT_ENGINE_REST_API_ACCESS_GROUP = "admin";
```

Comment:
The group 'admin' should be a group containing user administrating Flowable. I think it should be better to introduce a dedicated group for REST API access as `rest-api-access`, initiated with a dedicated user `rest-api-consumer`